knative-build/templates/build-bot-clusterrolebinding.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: knative-build-bot
+  name: {{ template "build.fullname" . }}-bot
```

Comment:
I've just been thinking about this again and wonder if we should postfix the helm release namespace in the cluster rolebinding name which means they will always be unique for each team. 
 We can still keep the `build.fullname` but adding the namespace means folks dont have to specify different build names in the `values.yaml`?

```
kind: ClusterRoleBinding
metadata:
  name: {{ template "build.fullname" . }}-{{ .Release.Namespace }}-bot
```

Reply:
/CC @garethjevans I think this approach would also fix the prow chart for the issue https://github.com/jenkins-x/jx/issues/2544